app/src/main/AndroidManifest.xml

```diff
@@ -7,18 +7,26 @@
     android:allowBackup="true"
     android:icon="@mipmap/ic_launcher"
     android:label="@string/app_name"
+    android:name="com.example.myapplication.OppiaApplication"
     android:roundIcon="@mipmap/ic_launcher_round"
     android:supportsRtl="true"
     android:theme="@style/OppiaTheme">
-    <activity android:name="org.oppia.app.HomeActivity">
+    <activity android:name=".HomeActivity">
+    </activity>
+    <activity android:name=".splash.SplashActivity"
+              android:configChanges="orientation|screenSize|keyboardHidden"
```

Comment:
Why do we want to override the default configuration handling for this activity?

Reply:
To avoid activity restart it was added. Now I have done changes to classes so removed this default configuration